spec/services/employee_finder_spec.rb

```diff
@@ -0,0 +1,35 @@
+require "rails_helper"
+
+describe EmployeeFinder do
+  describe "#employee_exists?" do
+    it "returns true if an employee exists in Slack" do
```

Comment:
Oh definitely - much better to be explicit rather than implicit so I agree!  Consider it done. :-)